### ISSUE TITLE
no dc:format for annotatable version

### ISSUE THREAD

**Issue by chrmor** [DM2E/dm2e-mappings#54]
http://data.dm2e.eu/data/rdf/resourcemap/ub-ffm/sammlungen/320380742_phys5001862/20140301004538912

**Comment by knepper** [comment]
My data are looking like this

``` xml

   <edm:WebResource rdf:about="http://sammlungen.ub.uni-frankfurt.de/msma/download/webcache/1000/5001862">
      <dc:format>image/jpeg</dc:format>
   </edm:WebResource>
```

How can I check this in pubby?

**Comment by knepper** [comment]
see #52

**Comment by kba** [comment]
Caused by #51

**Comment by kba** [comment]
Fixed by #51.
